tests/src/mods/utils.rs

```diff
@@ -141,3 +139,14 @@ pub fn query_dictionary_item(
     };
     builder.query(None, address, &empty_path)
 }
+
+pub fn get_privayte_keys() -> (
```

Comment:
Typo!

Reply:
typo === [🍰](https://emojipedia.org/shortcake/) for you ! thanks nice catch.